c/etcdserver/etcdserverpb/raft_internal.pb.go

```diff
@@ -2270,69 +2270,68 @@ var (
 func init() { proto.RegisterFile("raft_internal.proto", fileDescriptorRaftInternal) }
 
 var fileDescriptorRaftInternal = []byte{
```

Comment:
Why did this change? Same question for the other `.pb.go` files that changed.

Reply:
It's just the result of running protoc on the proto file in a new directory. 

File location is stored as part of [FileDescriptorProto](https://github.com/protocolbuffers/protobuf/blob/f5bdd7cd56aa86612e166706ed8ef139db06edf2/src/google/protobuf/descriptor.proto#L63) and after gziping it leads to a different payload.